chainio/clients/elcontracts/writer.go

```diff
@@ -545,3 +550,49 @@ func (w *ChainWriter) RegisterForOperatorSets(
 
 	return receipt, nil
 }
+
+func (w *ChainWriter) RemovePermission(ctx context.Context, request RemovePermissionRequest) error {
+	if w.permissionController == nil {
+		return errors.New("permission contract not provided")
+	}
+	noSendTxOpts, err := w.txMgr.GetNoSendTxOpts()
+	if err != nil {
+		return utils.WrapError("failed to get no send tx opts", err)
+	}
+
+	tx, err := w.permissionController.RemoveAppointee(
+		noSendTxOpts,
+		request.Account,
+		request.UserAddress,
+		request.Target,
+		request.Selector,
+	)
+	if err != nil {
+		return errors.New("call to permission controller failed: " + err.Error())
+	}
+	_, err = w.txMgr.Send(ctx, tx, request.WaitForReceipt)
+	return err
+}
+
+func (w *ChainWriter) SetPermission(ctx context.Context, request SetPermissionRequest) error {
```

Comment:
you need to send receipt back - so that you can populate transaction details on CLI. check the other write methods.